src/ssl.c

```diff
@@ -5424,8 +5422,16 @@ int wolfSSL_CertManagerEnableOCSPStapling(WOLFSSL_CERT_MANAGER* cm)
 
     return ret;
 }
-
-
+#if defined(SESSION_CERTS)
+STACK_OF(WOLFSSL_X509)* wolfSSL_get_peer_cert_chain(const WOLFSSL* ssl)
```

Comment:
Has this function been tested in an SSL/TLS connection to verify it returns the expected peer chain?

Reply:
Added session.chain->count value check via returned pointer.